### PR TITLE
Clockwork Marauders are more powerful but easier to kill

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: Clockwork Marauders no longer have Fatigue. It was difficult to balance and made them too easy to force into recalling. This means they just have health; they aren't forced to recall by anything, but can accordingly die much more easily.
rscadd: Accordingly Clockwork Marauders now have more health, do slightly more damage, block slightly more often, and have to go slightly further from their host to take damage.
rscadd: Marauders that are not recovering(from recalling while the host's health is too high to emerge) and are inside their host, or are within a tile of their host, will gradually heal their host until their host is above the health threshold to emerge.
/:cl:

Fatigue meant marauders basically had 100 health, but you couldn't KILL them unless they fought constantly for a minute, not including the time for them to lose fatigue.

It's more interesting for them to have a ton of health but actually have to MANAGE it to defend their host/fight people, and allowing them to actually heal their host if they're careful makes them better at defending.

### CODE REVIEW

**Comment by coiax** [comment]
This is a good time to re-suggest that we should have a generic HUD for health for simple animals. Can't we borrow some of the guardian UI stuff?

File: code/game/gamemodes/clock_cult/clock_mobs.dm [hunk 1]
```diff
@@ -129,33 +129,31 @@
 	name = "clockwork marauder"
 	desc = "A stalwart apparition of a soldier, blazing with crimson flames. It's armed with a gladius and shield."
 	icon_state = "clockwork_marauder"
-	health = 300 //Health is very high, and under most cases it will take enough fatigue to be forced to recall first
-	maxHealth = 300
+	health = 500
+	maxHealth = 500
```
Comment (KorPhaeron) [inline code/game/gamemodes/clock_cult/clock_mobs.dm]: Jesus

**Comment by jughu** [comment]
Could just make them holoparasites

**Comment by Mindustry** [comment]
I'm failing to see how this makes them "easier to kill" if you're buffing their HP, block chance, giving them higher damage, and allowing them to actively heal their master emerged or not.

**Comment by coiax** [comment]
Because previously, they took damage as "fatigue" damage, automatically disappearing if they took too much fatigue. Now if they don't recall themselves, they will actually just die, rather than being forced back into the host.

**Comment by ChangelingRain** [comment]
Basically they can super block for you but if they stick around and get hit too much, or you run away from them, they just fucking die.
They're definitely better, but that's because it's in comparison to something that had effectively 100 health but couldn't die without a ton of effort.

**Pull request merged**